charts/ia-case-api/values.yaml

```diff
@@ -7,6 +7,7 @@ java:
     POSTGRES_HOST: ia-case-api-postgres-db-{{ .Values.global.environment }}.postgres.database.azure.com
     POSTGRES_USERNAME: ia_case_api@ia-case-api-postgres-db-{{ .Values.global.environment }}
     POSTGRES_CONNECTION_OPTIONS: "?sslmode=require"
+    PROF_REF_DATA_URL: "http://rd-professional-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
```

Comment:
If we do this ... in theory we would not need the values in the vault as it sets the env correctly from global vars.

Reply:
as discussed, there is a need to refactor this and other urls which shouldn't be stored in Vault